packages/types/src/Metadata/util/testUtil.ts

```diff
@@ -20,6 +28,8 @@ export function decodeLatestSubstrate<Modules extends Codec> (
   latestSubstrate: object
 ) {
   it('decodes latest substrate properly', () => {
+    injectDefinitions();
```

Comment:
Not convinced this is the proper way to do this before the type checking tests. However mimics API, without doing magic in createType to just "pull these in".